### PR TITLE
update girder and osumo

### PR DESCRIPTION
**Pull request by opadron**
This update opens the doors for public users to create accounts and log
in using those account, and to run analysis with their data.  Currently,
however, one would still need to browse the girder UI to manage their
data sets.

This should be the final deployment update before the architectural
rework.  It will also probably be the last deployment stuck using Girder
1.0.

 - girder -> latest 1.x-maintenance branch
 - osumo -> latest master

### CODE REVIEW

**Comment by opadron** [comment]
Also note that after merging this PR, we should tag master as`2016-12-15`.

Edit: In light of the newly added task in the work plan, we should hold off on tagging master until we've merged this, and another PR for the go-to-girder link.

**Review by jeffbaumes — APPROVED** [review]

**Pull request merged**
